template_content/smart_contracts/README.md.jinja

```diff
@@ -0,0 +1,12 @@
+## How to add new smart contracts?
+
+By the default the template creates a single `HelloWorld` contract under {{ contract_name }} folder in the `smart_contracts` directory. To add a new contract:
+
+1. Create a new folder under `smart_contracts` directory and add define your new contract in `contract.py` file.
+2. Each contract has potentially has different creation parameters and deployment steps. Hence, you need to define your deployment logic in {% if deployment_language == 'python' %}`deploy_config.py`{% elif deployment_language == 'typescript' %}`deploy-config.ts`{% endif %}file.
+3. Reference your contract in `config.py` file. This will tell instruct the helper scripts on which exact contracts require building and generation of typed clients.
+{%- if deployment_language == 'typescript' %}
+4. Since you are generating a Typescript client, you also need to reference your contract in `index.ts` file. This will instruct the typescript specific helper scripts on how exactly to deploy your contract.
```

Comment:
```suggestion
4. Since you are generating a TypeScript client, you also need to reference your contract in `index.ts` file. This will instruct the TypeScript specific helper scripts on how exactly to deploy your contract.
```